### PR TITLE
Fix mysql issue #1557

### PR DESCRIPTION
**Pull request by trevorsibanda**
#1557

### CODE REVIEW

**Comment by AtkinsChang** [comment]
You should use ```new Config("MySQL", StandardTestDBs.MySQL, "MySQL", Seq("/dbs/mysql.sql")) { override testCode = ... }``` instead of putting test in `GeneratedCodeTest` which will cause compile error if database is not enabled.

**Comment by trevorsibanda** [comment]
@AtkinsChang thanks... had forgotten about that :)

**Comment by AtkinsChang** [comment]
I don't know why circleci's testing script doesn't set `-Dslick.testkit-config=test-dbs/testkit.travis.conf` to enable external db testing, maybe ask @cvogt

**Comment by cvogt** [comment]
@trevorsibanda test for this as well :)?

File: slick-testkit/src/codegen/scala/slick/test/codegen/GenerateMainSources.scala [hunk 1]
```diff
@@ -80,6 +80,17 @@ val  SimpleA = CustomTyping.SimpleA
       })
     },
     new UUIDConfig("CG10", StandardTestDBs.H2Mem, "H2Mem", Seq("/dbs/uuid-h2.sql")),
+    new Config("MySQL", StandardTestDBs.MySQL, "MySQL", Seq("/dbs/mysql.sql")){
```
Comment (trevorsibanda) [inline slick-testkit/src/codegen/scala/slick/test/codegen/GenerateMainSources.scala]: @cvogt Had added test

File: slick/src/main/scala/slick/jdbc/MySQLProfile.scala [hunk 2]
```diff
@@ -83,6 +83,7 @@ trait MySQLProfile extends JdbcProfile { profile =>
         case (v,"String")    => Some(Some(v))
         case ("1","Boolean") => Some(Some(true))
         case ("0","Boolean") => Some(Some(false))
+        case ( v , "scala.math.BigDecimal") => Some( Some( scala.math.BigDecimal(v) ) )
```
Comment (rbellamy) [inline slick/src/main/scala/slick/jdbc/MySQLProfile.scala]: I don't think you need this if PR #1551 is merged

**Pull request merged**